reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,168 @@ Explains language modes and their effect on PowerShell sessions.
 
 ## Long description
 
-The language mode of a PowerShell session determines, in part, which elements
-of the PowerShell language can be used in the session.
+The language mode of a PowerShell session determines which elements of the
+PowerShell language can be used in the session.
 
 PowerShell supports the following language modes:
 
-- **FullLanguage**
-- **ConstrainedLanguage** (introduced in PowerShell 3.0)
-- **RestrictedLanguage**
-- **NoLanguage**
+- `FullLanguage`
+- `RestrictedLanguage`
+- `ConstrainedLanguage` (introduced in PowerShell 3.0)
+- `NoLanguage`
 
-### What is a language mode?
+## What is a language mode?
 
 The language mode determines the language elements that are permitted in the
 session.
 
-The language mode is actually a property of the session configuration (or
-"endpoint") that is used to create the session. All sessions that use a
-particular session configuration have the language mode of the session
+The language mode is a property of the session configuration (or "endpoint")
+that's used to create the session. All sessions that use a particular session
+configuration have the language mode of the session configuration.
+
+All PowerShell sessions have a language mode. Remote sessions are created using
+the session configurations on the remote computer. The language mode set in the
+session configuration determines the language mode of the session. To specify
+the session configuration of a PSSession, use the **ConfigurationName**
+parameter of cmdlets that create a session.
+
+Beginning in PowerShell 7.3, you can run `pwsh` with the **ConfigurationFile**
+parameter. This allows you to start PowerShell using a specific configuration.
+
+## Finding the language mode of a session
+
+You can find the language mode of a `FullLanguage` or `ConstrainedLanguage`
+session by getting the value of the **LanguageMode** property of the session
+state.
+
+For example:
+
+```powershell
+$ExecutionContext.SessionState.LanguageMode
+ConstrainedLanguage
+```
+
+However, in sessions with `RestrictedLanguage` and `NoLanguage` modes, you
+can't use the dot method to get property values. Instead, the error message
+reveals the language mode.
+
+When you run the `$ExecutionContext.SessionState.LanguageMode` command in a
+`RestrictedLanguage` session, PowerShell returns the
+**PropertyReferenceNotSupportedInDataSection** and
+**VariableReferenceNotSupportedInDataSection** error messages.
+
+- **PropertyReferenceNotSupportedInDataSection**: Property references aren't
+  allowed in restricted language mode or a Data section.
+- **VariableReferenceNotSupportedInDataSection**: A variable that can't be
+  referenced in restricted language mode or a Data section is being
+  referenced.
```

Comment:
Should these be in blockquotes or wrapped as code to make it more clear that they're verbatim messages?

Reply:
Not necessary for this purpose. I would rather leave it as is for now. We have never had any confusion about it .